elements/pf-dropdown/pf-dropdown-item.css

```diff
@@ -0,0 +1,132 @@
+:host {
+  display: block;
+  width: 100%;
+  text-align: left;
+  white-space: nowrap;
+  border: none;
+  cursor: pointer;
+  font-size: var(
+    --pf-c-dropdown__menu-item--FontSize,
+    var(--pf-global--FontSize--md, 1rem)
+  );
+  font-weight: var(
+    --pf-c-dropdown__menu-item--FontWeight,
+    var(--pf-global--FontWeight--normal, 400)
+  );
+  line-height: var(
+    --pf-c-dropdown__menu-item--LineHeight,
+    var(--pf-global--LineHeight--md, 1.5)
+  );
+  color: var(
+    --pf-c-dropdown__menu-item--Color,
+    var(--pf-global--Color--dark-100, #151515)
+  );
+  background-color: var(
+    --pf-c-dropdown__menu-item--BackgroundColor,
+    transparent
+  );
+}
+
+:host li {
+  padding: var(
+      --pf-c-dropdown__menu-item--PaddingTop,
+      var(--pf-global--spacer--sm, 0.5rem)
+    )
+    var(
+      --pf-c-dropdown__menu-item--PaddingRight,
+      var(--pf-global--spacer--md, 1rem)
+    )
+    var(
+      --pf-c-dropdown__menu-item--PaddingBottom,
+      var(--pf-global--spacer--sm, 0.5rem)
+    )
+    var(
+      --pf-c-dropdown__menu-item--PaddingLeft,
+      var(--pf-global--spacer--md, 1rem)
+    );
+}
+
+[hidden] {
+  display: none !important;
+}
+
+:host([disabled]) {
+  pointer-events: none;
+  cursor: not-allowed;
+  --pf-c-dropdown__menu-item--Color: var(
+    --pf-c-dropdown__menu-item--disabled--Color,
+    var(--pf-global--Color--dark-200, #6a6e73)
+  );
+  --pf-c-dropdown__menu-item--BackgroundColor: var(
+    --pf-c-dropdown__menu-item--disabled--BackgroundColor,
+    transparent
+  );
+}
+
+:host(:hover) {
+  --pf-c-dropdown__menu-item--Color: var(
+    --pf-c-dropdown__menu-item--hover--Color,
+    var(--pf-global--Color--dark-100, #151515)
+  );
+  --pf-c-dropdown__menu-item--BackgroundColor: var(
+    --pf-c-dropdown__menu-item--hover--BackgroundColor,
+    var(--pf-global--BackgroundColor--light-300, #f0f0f0)
+  );
+  text-decoration: none;
+}
+
+hr {
+  --pf-c-divider--BorderWidth--base: var(--pf-global--BorderWidth--sm, 1px);
```

Comment:
The divider should have no padding or margin it appears from http://v4-archive.patternfly.org/v4/components/dropdown and also should not have a hover effect like the other dropdown items.  

![image](https://github.com/patternfly/patternfly-elements/assets/9708152/9bad019e-d528-4970-b903-5cdcc39015ce)